src/Components/Home/Users.js

```diff
@@ -74,7 +74,7 @@ function Users({ list }) {
       <div className="user-list flex flex-wrap justify-content-center">
         {!!filteredList &&
           filteredList.length > 0 &&
-          filteredList.map((user, key) => (
+          filteredList.slice(0, 25).map((user, key) => (
```

Comment:
What if you sliced it at 24? It looks nice that way. The see more functionality works well.

Reply:
Feel free to change it as you please sir,
;)